private/bufpkg/bufcheck/buflint/buflint_test.go

```diff
@@ -889,14 +883,9 @@ func TestCommentIgnoresOff(t *testing.T) {
 
 func TestCommentIgnoresOn(t *testing.T) {
 	t.Parallel()
-	testLintWithModifiers(
```

Comment:
Same, there should be no diff here.

Reply:
resolved in [1b2c71c](https://github.com/bufbuild/buf/pull/2377/commits/1b2c71cceeb54702ea58bc69970a1a434fb6410e)